[DOC-562] Update build the docs page with instructions for Ubuntu and a specific hugo version. (#25121)
yugabyte/yugabyte-db: docs/content/preview/contribute/docs/macos.md

<!--
+++
private=true
+++
-->

* **Command-line tools for Xcode** on macOS.

    ```sh
    $ xcode-select --install
    ```

    Xcode is many gigabytes. Install the command-line tools unless you actually need the full Xcode.

* [**Homebrew**](https://brew.sh) on macOS.

* **Python** (version 3.10 or earlier): You need `python` to be available somewhere in your shell path.

    Recent versions of macOS have only a `python3` executable, as does the Homebrew install. You can use [pyenv](https://github.com/pyenv/pyenv) to manage multiple versions of python on your system. Make sure to point to Python version 3.10 or earlier.

> * **Hugo**: `brew install hugo` installs the latest version.

Reviewer: So for MacOS, add these instructions to install a specific version. Lets fix 0.125.6 as thats the current one we are using :
Include the following steps to install a specific hugo version. (0.125.6)
• Unpin hugo - brew unpin hugo
• Uninstall older version - brew uninstall hugo
• Download the attached file.
• brew install hugo.rb
• Prevent automatic updates with brew pin hugo

Author: Added the steps for v0.131.0 version of Hugo. Please check.

<!--
+++
private=true
+++
-->

* **Command-line tools for Xcode** on macOS.

    ```sh
    $ xcode-select --install
    ```

    Xcode is many gigabytes. Install the command-line tools unless you actually need the full Xcode.

* [**Homebrew**](https://brew.sh) on macOS.

> * **Python** (version 3.10 or earlier): You need `python` to be available somewhere in your shell path.

Reviewer 2: ```suggestion
* **Python** (v3.10 or earlier): You need `python` to be available somewhere in your shell path.
```

>     Recent versions of macOS have only a `python3` executable, as does the Homebrew install. You can use [pyenv](https://github.com/pyenv/pyenv) to manage multiple versions of python on your system. Make sure to point to Python version 3.10 or earlier.

Reviewer 2: ```suggestion
    Recent versions of macOS have only a `python3` executable, as does the Homebrew install. You can use [pyenv](https://github.com/pyenv/pyenv) to manage multiple versions of python on your system. Make sure to point to Python v3.10 or earlier.
```

> * **Hugo**: Install Hugo `v0.131.0` version. Follow these steps:

Reviewer 2: stick with the current one v0.125.6

Author: ``` suggestion
* **Hugo**: Install Hugo `v0.125.6` version. Follow these steps:
```

> * **Hugo**: Install Hugo `v0.125.6` version. Follow these steps:

Reviewer 2: ```suggestion
* **Hugo**: Install Hugo v0.125.6. Follow these steps:
```

  * Unpin Hugo to stop its formula from being updated - `brew unpin hugo`
  * Uninstall any older version if installed - `brew uninstall hugo`

>   * Download the `v0.131.0` formula file from [Homebrew's repository](https://github.com/Homebrew/homebrew-core/blob/69dfbc7747a766d9e7e2a94d57adacae359d8b5a/Formula/h/hugo.rb).

Reviewer 2: v0.125.6

Author: ```suggestion
  * Download the `v0.125.6` formula file from [Homebrew's repository](https://github.com/Homebrew/homebrew-core/blob/d6cdfadac69489a73e65b1ef46f59382836219c4/Formula/h/hugo.rb).
```

>   * Download the `v0.125.6` formula file from [Homebrew's repository](https://github.com/Homebrew/homebrew-core/blob/d6cdfadac69489a73e65b1ef46f59382836219c4/Formula/h/hugo.rb).

Reviewer 2: ```suggestion
  * Download the v0.125.6 formula file from [Homebrew's repository](https://github.com/Homebrew/homebrew-core/blob/d6cdfadac69489a73e65b1ef46f59382836219c4/Formula/h/hugo.rb).
```
